netci.groovy

```diff
@@ -641,8 +338,13 @@ def supportedFullCyclePlatforms = ['Windows_NT', 'Ubuntu14.04', 'OSX']
             Utilities.addArchival(newJob, archiveContents)
             // Set up triggers
             if (isPR) {
-                // Set PR trigger.
-                Utilities.addGithubPRTriggerForBranch(newJob, branch, "Innerloop ${os} ${configurationGroup} Build and Test")
+                // Set PR trigger, we run Windows_NT, Ubuntu 14.04, CentOS 7.1 and OSX on every PR.
+                if ( os == 'Windows_NT' || os == 'Ubuntu14.04' || os == 'Centos7.1', os == 'OSX' )
```

Comment:
Cool!  They're not documented, by I've used several syntax error operators in C# and other languages in the past.  They come in quite handy when you're looking for reasons to grumble, frown, and become otherwise agitated.